ondewo/t2s/custom_phonemizer.proto

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 ONDEWO GmbH
+// Licensed under the ONDEWO GmbH license, Version 1.0 (the "License");
+// you must not use this file except in compliance with the License.
+// You must obtain a copy of the License at
+// office@ondewo.com
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package ondewo.t2s;
+import "google/protobuf/empty.proto";
+
+// endpoints of custom phonemizer
+service CustomPhonemizers{
+    rpc GetCustomPhonemizer (PhonemizerId) returns (CustomPhonemizerProto) {
+    };
+    rpc CreateCustomPhonemizer (CreateCustomPhonemizerRequest) returns (PhonemizerId) {
+    };
+    rpc DeleteCustomPhonemizer (PhonemizerId) returns (google.protobuf.Empty) {
+    };
+    rpc UpdateCustomPhonemizer (UpdateCustomPhonemizerRequest) returns (CustomPhonemizerProto) {
```

Comment:
why does `UpdateCustomPhonemizer` return `CustomPhonemizerProto`, while `CreateCustomPhonemizer` returns only `PhonemizerId`?

Reply:
> why does `UpdateCustomPhonemizer` return `CustomPhonemizerProto`, while `CreateCustomPhonemizer` returns only `PhonemizerId`?

When you create new phenemizer you send it to the server, so the only new thing server does is create new id. Therefore there is no need to send the whole phonemizer back, client has it already, except id, that will be sent. On the other hand, if you update phonemizer, depending on the update mode you get a phonemizer that client does not have yet - you just add some new words for example. So it make sense to send it back.